ambry-cloud/src/main/java/com.github.ambry.cloud/azure/AzureCloudDestination.java

```diff
@@ -245,6 +256,24 @@ public boolean updateBlobExpiration(BlobId blobId, long expirationTime) throws C
     return updateBlobMetadata(blobId, CloudBlobMetadata.FIELD_EXPIRATION_TIME, expirationTime);
   }
 
+  //@Override
+  public CloudBlobMetadata getBlob(String partitionPath, String blobFileName, OutputStream outputStream)
+      throws CloudStorageException {
+    try {
+      String containerName = getAzureContainerName(partitionPath);
+      CloudBlobContainer azureContainer = azureBlobClient.getContainerReference(containerName);
+      CloudBlockBlob azureBlob = azureContainer.getBlockBlobReference(blobFileName);
+      if (!azureBlob.exists(null, null, blobOpContext)) {
+        return null;
+      }
+      azureBlob.download(outputStream, null, null, blobOpContext);
+      azureBlob.getMetadata(); // convert to CBM
+      return null;
```

Comment:
So this method always returns null? What's its purpose?

Reply:
Sorry, this was temporary unfinished code that was not meant for this PR.  I'll remove it.